### ISSUE TITLE
Update Browser Support

### ISSUE THREAD

**Issue by Nevraeka** [webcomponents/webcomponents.github.io#329]
- With Microsoft deprecating updates for IE9 & IE10 should we support IE10+ or IE11 only?
- Also should we should mobile browser support here as well?

**Comment by Nevraeka** [comment]
cc @robdodson @csuwildcat

**Comment by robdodson** [comment]
I'm not sure what the color codes mean but I think everything that is currently red for custom elements / shadow dom should be set to yellow since everyone is either actively working on those and some folks have already shipped them in preview.

I think HTML Imports should be the only red squares.

Mobile browser support would be cool to include. Might make the table kinda long though?

**Comment by Nevraeka** [comment]
@robdodson I agree on the mobile browsers. I was thinking we could either make the images smaller and have 2 rows or just say we support the latest mobile versions of these as well as the desktop versions

As for yellow on Shadow DOM - are we sure about active development in all browsers or has it been scheduled but work has not started yet ?

**Comment by robdodson** [comment]
In Safari I'm pretty sure they're actively prototyping Custom Elements (there's a ticket...somewhere) and Shadow DOM v1 just shipped in Safari Technology Preview.

As for Edge... @csuwildcat any updates on that?

**Comment by Nevraeka** [comment]
What about IE10 support? Is that still necessary? I would think so but IDK

**Comment by csuwildcat** [comment]
@robdodson it is in the queue, but behind Service Worker development - they're just a bit backed up at the moment.

**Comment by robdodson** [comment]
@Nevraeka I don't think there's any plan to work on IE9-10 support in either webcomponentsjs or polymer.

**Comment by csuwildcat** [comment]
@robdodson IE9-10 combine for almost 10% of global market share, from my quick look - which seems rather significant. Web Components JS doesn't offer support for browsers that low, but we added a couple small API polyfills to our distribution that enable it, and we haven't had issues (with Custom Elements, not Shadow DOM)

**Comment by robdodson** [comment]
oh sorry, not saying it's not an important sector, I just don't think anyone (on our end at least) has the availability to work on supporting it.

> we added a couple small API polyfills to our distribution that enable it, and we haven't had issues (with Custom Elements, not Shadow DOM)

Related to x-tag?

**Comment by csuwildcat** [comment]
@robdodson yeah, we get IE9-10 support for the non-Shadow DOM specs just by including a DOMTokenList polyfill

**Comment by robdodson** [comment]
@csuwildcat would you want to vend those custom element polyfills on webcomponents.org with instructions on how to use them?

**Comment by Nevraeka** [comment]
:+1 to 

> @csuwildcat would you want to vend those custom element polyfills on webcomponents.org with instructions on how to use them?

good call @robdodson . I want to update the resources page asap anyway and thats a great place for these

**Comment by Nevraeka** [comment]
From a Shadow DOM perspective @treyshugart (SkateJS) & @eisenbergeffect (Aurelia) both have polyfills for named-slots for Shadow DOM v1 . It might make sense to work together toward a common project on these polyfills. Thoughts @csuwildcat or @robdodson ?

**Comment by csuwildcat** [comment]
I suppose the first step would be to setup a shared set of tests that each
implementation can run against the same criteria. @robdodson, is there a
current set of tests that are easy to run against other polyfill
implementations?

**Comment by robdodson** [comment]
Good question. @azakus @addyosmani do you know of any SD unit test suite?

**Comment by treshugart** [comment]
We have a set of tests for the DOM API that @nadiam84 has been doing quite a bit of work on but it's not exhaustive.

FWIW, our polyfill is a polygap / prollyfill. We only intend on supporting the JavaScript API as that is a must-have for integration with React and other virtual DOM libraries and other parts that we can keep performant but CSS support is a nice to have for us and isn't worth the page weight and performance hit. IE9 is supported. 

Happy to do anything we can to collaborate on this stuff.

**Comment by robdodson** [comment]
It looks like there's a number of tests living in the webcomponents.js repo that use WebComponentTester
https://github.com/webcomponents/webcomponentsjs/tree/master/tests

**Comment by miguelamello** [comment]
Polymer 1.0 is not working properly in IE 11.0.9600.18314
Many average users still use IE 11, and even IE10. 
I think it is a concern.

**Comment by Nevraeka** [comment]
Maybe it makes sense to list the browser support for native as we do.

I think it would be great to break down the polyfill browser support as well since more polyfills are being used now as well as versions and pieces of webcomponents.js. Breaking out the parts of webcomponents.js & webcomponents-lite.js  and the support makes sense. Also taking any other polyfills and doing the same.

**Comment by teppeis** [comment]
Why is HTML Imports of Firefox YELLOW?
Why is "IE/Edge" combined?
